pkgs/by-name/li/limine/package.nix

```diff
@@ -0,0 +1,61 @@
+# Builds limine with all available features.
+
+{
+  # Helpers
+  fetchurl
+, lib
+, stdenvNoCC
+, # Dependencies
+  llvmPackages
+, gnumake
+, mtools
+, nasm
+}:
+
+let
+  version = "7.1.0";
+  # It is recommended by limine to use the release tarballs instead the git repo.
+  limineSrc = fetchurl {
+    url = "https://github.com/limine-bootloader/limine/releases/download/v${version}/limine-${version}.tar.gz";
+    sha256 = "sha256-oaQA461MGVuG7aJri2HyYb6Pu8BXOWTF3Wq/SputQSw=";
+  };
+in
+stdenvNoCC.mkDerivation {
+  inherit version;
+  pname = "limine";
+  src = limineSrc;
+  nativeBuildInputs = [
+    gnumake
+    llvmPackages.bintools
+    llvmPackages.clang
+    llvmPackages.lld
+    mtools
+    nasm
+  ];
+  doCheck = false;
+  configurePhase = ''
+    ./configure --enable-all
+  '';
+  buildPhase = ''
+    make -j $(nproc)
+  '';
+  installPhase = ''
+    cp -R bin/. $out
+  '';
```

Comment:
Have you tried the default `install` target? There Makefile seems to do the right thing. This would also install man pages and development headers.

The `bin/limine` tool seems to refer to the EFI files anyhow. That means they also need to be installed (which is what the normal install target does).

tl;dr

Remove the manual `installPhase`. 

Add `outputs = [ "out" "doc" "dev" "man" ];` to only install what people need.

Reply:
> outputs = [ "out" "doc" "dev" "man" ];

love it, thanks!